conda_metadata_app/app_config.py

```diff
@@ -397,6 +397,12 @@ def validate_channels(cls, channels: dict[str, Channel]) -> dict[str, Channel]:
     each artifact.
     """
 
+    render_dependencies_as_table: bool = False
```

Comment:
Maybe call this `render_dependencies_as_table_default` instead and explain in the docstring that this setting can be overridden by the user?